docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -42,7 +42,7 @@ Next, expand the template into multiple files, one for each item to be processed
 $ mkdir ./jobs
 $ for i in apple banana cherry
 do
-  cat job.yaml.txt | sed "s/\$ITEM/$i/" > ./jobs/job-$i.yaml
+  cat job.yaml | sed "s/\$ITEM/$i/" > ./jobs/job-$i.yaml
```

Comment:
These changes seem very orthogonal to the rest of the comment, did you intend to  include them?